### PR TITLE
clarify each tools purpose

### PR DESCRIPTION
**Pull request by nanliu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,30 +42,48 @@ For more info see:
...
-Update all plugins (must use the noop option for now):
+The repository provides several Snap plugin maintenance tools:
+
+* plugin repo sync: this updates repos with the latest license, testing, and travis ci configs
+* generate travis ci secret: create secret token for publishing binaries
+* catalog metadata: update plugin_catalog and github io page with latest github plugin metadata
+
+### Update Snap plugin repository
+
+To update all plugin repositories (must use the noop option for now):
```
Comment (rashmigottipati) [inline README.md]: Is this section used only when we need to update .sync.yml?

Comment (nanliu) [inline README.md]: The global update is typically when we make changes to this repo, such as changing go version from 1.7.3 -> 1.7.4. I've added some common use cases on when we run global updates vs. individual updates.

**Comment by kindermoumoute** [comment]
Maybe it would be easier to use the readme with an index, what you think?

**Comment by nanliu** [comment]
@kindermoumoute, I added a toc per your suggestion. Seems sensible given the length of this doc.

**Pull request merged**